src/libs/nuked/opl3.c

```diff
@@ -1280,6 +1281,7 @@ void OPL3_Reset(opl3_chip *chip, uint32_t samplerate)
         chip->slot[local_ch_slot + 3].channel = channel;
         if ((channum % 9) < 3)
         {
+            assert(channum <= 11);
```

Comment:
I'm not quite sure where 11 is coming from.

Reply:
Will replace it with a constexpr calculation. (Unfortunately the baseline code doesn't define constants.. they use magic values everywhere, but I'll drop a couple in).